mri/operators/linear/wavelet.py

```diff
@@ -66,6 +68,11 @@ def __init__(self, wavelet_name, nb_scale=4, verbose=0, dim=2,
         n_proc = self.n_jobs
         if n_proc < 0:
             n_proc = joblib.cpu_count() + self.n_jobs + 1
+        if n_proc > 0:
+            if wavelet_name in wavelist()['isap-2d'] or wavelet_name in wavelist()['isap-3d']:
+                warnings.warn("n_jobs is currently unsupported for ISAP wavelets, setting n_jobs=1")
+                self.n_jobs = 1
+                n_proc = 1
         # Create transform queue for parallel execution
```

Comment:
I have removed support for n_jobs in ISAP wavelets as they are erratic in behaviour and crash randomly sometimes. 
We can get back at this once it is fixed in https://github.com/CEA-COSMIC/pysap/issues/101

Reply:
hmmm it's weird.
Can you document this (i.e. write the traceback and minimal reproducing example) in the issue?

Reply:
I have added the both in the pysap issue earlier